packages/@ember/-internals/glimmer/tests/integration/components/curly-components-test.js

```diff
@@ -2576,6 +2576,44 @@ moduleFor(
       }, expectedBacktrackingMessage);
     }
 
+    ["@test when a shared dependency is changed during children's rendering (tracked)"](assert) {
+      if (!EMBER_METAL_TRACKED_PROPERTIES) {
+        assert.expect(0);
+        return;
+      }
```

Comment:
There is a rebase conflict (likely from removing the tracked properties feature flags), would you mind fixing it up?